wallet/wallet.go

```diff
@@ -2722,22 +2724,27 @@ func (w *Wallet) ListLeasedOutputs() ([]*ListLeasedOutputResult, error) {
 	err := walletdb.View(w.db, func(tx walletdb.ReadTx) error {
 		ns := tx.ReadBucket(wtxmgrNamespaceKey)
 		outputs, err := w.TxStore.ListLockedOutputs(ns)
+		if err != nil {
+			return err
+		}
 
 		for _, output := range outputs {
 			details, err := w.TxStore.TxDetails(ns, &output.Outpoint.Hash)
 			if err != nil {
 				return err
 			}
+			value := details.MsgTx.TxOut[output.Outpoint.Index].Value
```

Comment:
Why not just `txOut := details.MsgTx.TxOut[output.Outpoint.Index]` and then below `Value: txOut.Value`...?
Also, empty lines are nice to get some "breathing room". They should be used to group statements together logically. So I'd suggest adding one after the `if err != nil { ... }` block and before `results = append(results, result)`.

Reply:
I agree with both points. Sorry for making silly mistakes 🤦🏻‍♂️

Reply:
No worries ;-)